src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        ColorEnum[] colors = ColorEnum.values();
+        Random random = new Random();
```

Comment:
Think about which variables should be local in the method and which should be class-level

`Random class` instantiates every time the method is invoked.
Should be instantiated once and reused multiple times.